.github/workflows/update-dist.yml

```
@@ -0,0 +1,35 @@
+# When Dependabot updates dependency, this workflow synchronize the change
+# to the /dist dir to let users enjoy new feature without manual sync.
+name: Update the dist dir
+on:
+  pull_request_target:
+    types:
+      - labeled # for initial execution
+      - synchronize # to support `@dependabot recreate` command
+jobs:
+  run-ncc:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'dependencies')
```

I think this would be safer so that this isn't some sort of attack vector. I don't think someone can change the `github.actor` to match dependabot.
```suggestion
    if: ${{ github.actor == 'dependabot[bot]' }}
```